### PR TITLE
Feature/carousel functionality version2

### PR DESCRIPTION
**Pull request by ReinisZogots**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,43 +1,85 @@
...
-    current.classList.remove('move-left-next-slide');
-    current.classList.remove('visible');
-    if(current.nextElementSibling){
-        current.nextElementSibling.classList.add('move-left-next-slide');
-        current.nextElementSibling.classList.add('visible');
+next.addEventListener('click', function() {
+    if(direction === 1) {
+        direction = -1;
+        slider.prepend(slider.lastElementChild);
+        carousel.style.justifyContent = 'flex-start';
```
Comment (VSlobolinskyi) [inline js/main.js:11]: Don't use inline styles. If you need to add or remove some property toggle class. If it's not possible then you will have to re-think your logic.

File: js/main.js [hunk 2]
```diff
@@ -1,43 +1,85 @@
...
+        slider.style.transform = 'translate(-33.3333%)';
+    } else {
+        direction = -1;
+        carousel.style.justifyContent = 'flex-start';
+        slider.style.transform = 'translate(-33.3333%)';
```
Comment (VSlobolinskyi) [inline js/main.js:16]: Class

File: js/main.js [hunk 3]
```diff
@@ -1,43 +1,85 @@
...
+    if(direction === -1) {
+        slider.appendChild(slider.firstElementChild);
+        direction = 1;
     }
-    else {
-        slides[slides.length -1].classList.add('move-right-prev-slide');
-        slides[0].classList.remove('move-left-next-slide');
-        slides[slides.length -1].classList.add('visible');
+    carousel.style.justifyContent = 'flex-end';
+    slider.style.transform = 'translate(33.3333%)';
```
Comment (VSlobolinskyi) [inline js/main.js]: class

File: js/main.js [hunk 4]
```diff
@@ -1,43 +1,85 @@
...
+        slider.appendChild(slider.firstElementChild);
+    } else {
+        slider.prepend(slider.lastElementChild);
     }
-}
+    
+    slider.style.transition = 'none';
+    slider.style.transform = 'translate(0)';
+    setTimeout(function(){
+        slider.style.transition = 'all 0.5s';
```
Comment (VSlobolinskyi) [inline js/main.js:45]: class

**Review by VSlobolinskyi — CHANGES_REQUESTED** [review]

File: css/style.css [hunk 5]
```diff
@@ -79,20 +79,20 @@ body {
...
     display: flex;
     justify-content: flex-start;
-    /* overflow: hidden; */
+    overflow: hidden;
 }
 
 .carousel__item-container {
     display: flex;
     height: 100%;
     width: 300%;
```
Comment (VSlobolinskyi) [inline css/style.css:93]: Why do you have with: 300%?

Comment (ReinisZogots) [reply css/style.css:93]: because there are 3  slides

**Review by VSlobolinskyi — APPROVED** [review]

**Pull request merged**